src/__tests__/data/ColumnWithDefaultExportOnly.tsx

```diff
@@ -19,7 +19,10 @@ export interface IColumnProps {
 /**
  * ColumnWithDefaultExportOnly description
  */
-export default class Column extends React.Component<IColumnProps, {}> {
+export default class ColumnWithDefaultExportOnly extends React.Component<
```

Comment:
Is renaming `Column` to `ColumnWithDefaultExportOnly` really enough for testing your implementation? 

I mean if the existing test before used the file name which is `ColumnWithDefaultExportOnly` your changed test has to passed before as well or am I wrong?